plugins/eddn.py

```diff
@@ -226,15 +237,31 @@ def send(self, cmdr: str, msg: Mapping[str, Any]) -> None:
                 logger.trace_if('plugin.eddn', "EDDN is still objecting to empty commodities data")
                 return  # We want to silence warnings otherwise
 
+            from base64 import b64encode  # we dont need this to be around until this point, which may never hit
+            if r.status_code == 413:
+                logger.debug(dedent(
+                    f"""\
```

Comment:
`"""` or `'''` (line 257 below), pick one.  Our standard is currently `'''` by default IIRC.  Although I switch to `"` if it's more convenient due to `'` in the strings.  But at least be consistent within this PR :)

Reply:
for `'''` `'` wont matter  -- sorry habit from switching between some codebases. Either way, fixed